code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -51,8 +51,14 @@
 					reagent_volumes[T] = min(reagent_volumes[T] + 5, volume)
 	return 1
 
-/obj/item/weapon/reagent_containers/borghypo/attack(var/mob/living/M, var/mob/user)
+/obj/item/weapon/reagent_containers/borghypo/afterattack(var/obj/target, var/mob/user, proximity)
+
+	if(!proximity)
+		return
+
+	var/mob/living/M = target

```

Comment:
Why this change? The initial argument was fine.

Reply:
Question, if afterattack target was casted to a mob type, would it run at all if the target is actually a non-mob? I want it to run the parent afterattack if the target is not a mob.